client/components/common/ReportStatusSummary/index.jsx

```diff
@@ -32,6 +32,17 @@ const ReportStatusSummary = ({
 
     const conflictsCount = metrics.conflictsCount ?? 0;
     const percentComplete = calculatePercentComplete(testPlanReport);
+    const conflictsEl =
+      conflictsCount > 0 ? (
+        <a
+          style={{ color: 'salmon' }}
+          href={`/test-queue/${testPlanReport.id}/conflicts`}
+        >
+          with {conflictsCount} conflicts
+        </a>
+      ) : (
+        `with ${conflictsCount} conflicts`
```

Comment:
Interesting. I left it there by default since it exists currently but I am in favor of removing text when possible and reducing the cognitive complexity of viewing the app. Changes here [c345507](https://github.com/w3c/aria-at-app/pull/1195/commits/c345507665254ff10721f2891028b287f07cb68a)